save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -105,6 +122,29 @@ class DownloadFilesController(
         @PathVariable projectName: String,
     ): Mono<ByteBufferFluxResponse> = downloadByFileKey(fileInfo.toStorageKey(), organizationName, projectName)
 
+    @PostMapping(path = ["/api/$v1/files/download"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
```

Comment:
do we need it as API? maybe move to `/internal`?

Reply:
Already changed in 84efeba